### DISCUSSION TITLE
VBET 14060004 blurry

### DISCUSSION THREAD

**Discussion by nbouwes** [Riverscapes/riverscapes-website-jekyll#24] [Riverscapes Data Exchange] [answered]
Not sure I am posting this in the right place....

Not a big deal as this old VBET run will be soon replaced but for some reason this comes in quite blurry. Thought you might like to know

https://webrave.riverscapes.xyz/#/Anabranch/7d66b4d0-aa7b-4e74-879f-9954e08e2395

**Comment by joewheaton** [comment, marked as answer]
This is perfect place to post @nbouwes.  Thanks. For now, @MattReimer and I agreed to cap upper zoom level at 15 (you can zoom up to 20). I think the blurriness you're seeing is explained in the warning below (if not let me know).
![image](https://user-images.githubusercontent.com/9723905/142273539-7c92a19b-a516-4b88-98c4-06c5ecb1c667.png)

If you take that project and look at it in QRAVE or ArcRAVE, you will see full resolution. It is computationally expensive to produce tiles for all these zoom levels and for most of our national level projects 15 is adequate. However, @MattReimer and I have discussed making this higher for higher resolution rasters (e.g. CHaMP 0.1 m rasters or Lidar 0.5 - 2.0 m). It requires some logic that doesn't currently exist for warehouse, so we have not gotten to it. Any input you have here is helpful @nbouwes.

FYI @jtgilbert keep an eye on this discussion board for posts from users like this.

**Reply by joewheaton** [reply]
@MattReimer any thoughts?

**Reply by MattReimer** [reply]
Right now we're really stretching the system to even get the resolution we've got.  Sure, 14060004 is a bit blurry but I can make out the pixels and that's kind of the best we can do for now on the web, given the amount of tiles we're throwing around. 

If we want the web to resolve differently we need either a different technology and/or variable logic to read out the cell resolution and calculated max/min bounds on the fly.

**Reply by nbouwes** [reply]
Gotta give it more Scottie, damn it!, the federation of riverscapes depends on this!

It is what it is, fine with me, I was just providing feedback based on how clear everything else seems to be.

**Reply by joewheaton** [reply]
Keep the feedback coming @nbouwes. We need it. Thanks.

**Reply by MattReimer** [reply]
Absolutely. We need this kind of feedback to figure out where the priorities are for things like new features and improvements. All good to hear.
